gitlab/cloud-config.yaml

```diff
@@ -21,6 +21,7 @@ runcmd:
     - 'echo "postfix postfix/protocols       select  all" | sudo debconf-set-selections'
     - 'apt-get install postfix -y'
     - 'curl https://packages.gitlab.com/install/repositories/gitlab/gitlab-ee/script.deb.sh | sudo bash'
+    - "export GITLAB_ROOT_PASSWORD={{ params.gitlabrootpassword | escape.shell }}"
```

Comment:
I couldn't get this to work using `'`, seems like `escape.shell` uses it. That's why this line is using `"`